text.ipynb

```diff
@@ -535,7 +535,7 @@
     "\n",
     "`[$][0-9]+([.][0-9][0-9])?`\n",
     "\n",
-    "Where `+` means 1 or more occurrences and `?` means at most 1 occurrence. Usually a template consists of a prefix, a target and a postfix regex. In this template, the prefix regex can be \"price:\", the target regex can be the above regex and the postfix regex can be empty.\n",
+    "Where `+` means 1 or more occurrences and `?` means atmost 1 occurrence. Usually a template consists of a prefix, a target and a postfix regex. In this template, the prefix regex can be \"price:\", the target regex can be the above regex and the postfix regex can be empty.\n",
```

Comment:
'atmost' should be two words: 'at most' was correct.

Reply:
If you say so, @MrDupin :) thank you for pointing that out